packages/react-ui/src/widgets/PieWidgetUI/PieWidgetUI.js

```diff
@@ -71,7 +71,11 @@ function PieWidgetUI({
   const _percentFormatter = useMemo(
     () =>
       percentFormatter ||
-      ((v) => `${intl.formatNumber(v, { maximumFractionDigits: 2 })}%`),
+      ((v) =>
```

Comment:
As this has caused problems with e2e, please double-check the case when `total === 0`, it should returns `0,00%`

It was also modified in your last PR -> https://github.com/CartoDB/carto-react/pull/844/files#diff-aef7c79c208e732a247556c69887e48f3f8b2076d3bf28aeeb9abde1419806adL98-L99

Reply:
Checked